.github/workflows/tests.yml

```diff
@@ -35,7 +37,7 @@ jobs:
         python -m pip install -r requirements.txt --ignore-installed apache-airflow==${{ matrix.airflow-version }}
     - name: Run unit tests
       run: |
-        python -m unittest discover -s tests/unit
+        make unit
```

Comment:
Hi @r3b-fish I have been working on the plugin recently and found that `Makefile` looks to conflict with GitHub Action config. You have placed `make unit` as a command to run the tests. According to the `Makefile` `unit` has the following chain of dependencies: `unit -> venv -> $(VENV)/bin/activate`:

https://github.com/lithiumlabcompany/airflow-clickhouse-plugin/blob/adf4b43f6216c2e7b694cbfc671eb33c7993f3d6/Makefile#L25-L26

https://github.com/lithiumlabcompany/airflow-clickhouse-plugin/blob/adf4b43f6216c2e7b694cbfc671eb33c7993f3d6/Makefile#L13

https://github.com/lithiumlabcompany/airflow-clickhouse-plugin/blob/adf4b43f6216c2e7b694cbfc671eb33c7993f3d6/Makefile#L7-L10

The last target runs `pip install -r requirements.txt` and `pip install pandas`. Does it mean that it literally installs the packages again (after they are already installed by GitHub Action step `Install dependencies`) on each run and also always installs `pandas`? Or are these steps not executed for some reason by `make`?

This behaviour looks to always install the latest version and pandas for each test regardless of Airflow version specified by the version matrix of the GitHub action. Please confirm my understanding here.

Reply:
Hey Anton!

I think we should use isolated environment to run tests without artifacts that include into github/actions. I don't have good experience with github actions but may we use a think image to run tests?

Also, I'll try to run docker image and reproduce behaviour on local machine and return to you with info or PR.

Reply:
Could you please confirm, that when we run `make unit`, it runs `$(VENV)/bin/activate` target?

Reply:
I have checked the recent run and yes, it installs the latest Airflow version regardless of the versions matrix: https://github.com/bryzgaloff/airflow-clickhouse-plugin/actions/runs/5852899673/job/15865902746?pr=67#step:5:17

I plan to remove Makefile from the repository as it introduces an implicit behaviour to the parts where we need to avoid this. Please keep Makefile as a part of your local development environment: it is not widely adopted by Python users.

And of course, you will remain in the list of contributors :) Thank you for your contribution anyway 🤝

Reply:
Sure! Also you can add --ignore-installed option to fix it.